net/FlatBuffers/StructPos.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace FlatBuffers 
+{
+    public struct StructPos 
+    {
+        public StructPos(int bb_pos, ByteBuffer bb) 
+        {
+            this.bb_pos = bb_pos;
+            this.bb = bb;
+        }
+
+        public readonly int bb_pos;
```

Comment:
If we're changing this, we should consider changing the code to be more C# formatting compliant - eg: public fields are in PascalCase. The reason is that it will trip a number of static/style analyzers.